packages/devtools_server/lib/src/chrome.dart

```diff
@@ -71,13 +71,11 @@ class Chrome {
   /// Each url in [urls] will be loaded in a separate tab.
   static Future<void> start(
     List<String> urls, {
-    List<String> args,
+    List<String> args = const [],
     int port,
   }) async {
-    args ??= [];
-    args.addAll(urls);
-
-    await Process.start(_executable, args);
+    final processArgs = List.from(urls)..addAll(args);
```

Comment:
you switched the order of the urls and the args.
I would write
args.toList()..addAll(urls)

Reply:
done.